modules/ha/pages/elastic-cluster.adoc

```diff
@@ -0,0 +1,33 @@
+= Compute-on-demand Elastic Cluster
+:description: Overview of TigerGraph's compute-on-demand ER cluster.
+
+TigerGraph allows you to spin up an Elastic Read-only (ER) cluster from your primary cluster to handle compute-intensive online analytical processing (OLAP) queries on demand.
+Operations on the ER cluster do not interfere with the primary cluster in any way.
+
+NOTE: This feature is currently in alpha.
```

Comment:
Let us not use the alpha term here. Instead, use "preview version" for the early adopters/customers

Reply:
Yes, I think I had already changed to "preview version" as I realized that alpha is an inappropriate term